auth-foundation/src/main/java/com/okta/authfoundation/credential/Token.kt

```diff
@@ -187,3 +234,8 @@ internal class SerializableToken internal constructor(
         )
     }
 }
+
+@Serializable
+private class TokenIssuedAtPayload(
```

Comment:
I'd suggest renaming this to something to signify why it's being used.  In my mind it feels like this, and the `getIssuedTime` function, are there to extract values from the server response necessary for storage or validation of the token.  So `TokenInitializationPayload` or something?

Reply:
I've renamed it to TokenInitializationPayload